bql/planner/planner_test.go

```diff
@@ -763,30 +767,30 @@ func TestPlannerQuery(t *testing.T) {
 
 		st := &semantic.Statement{}
 		if err := p.Parse(grammar.NewLLk(entry.q, 1), st); err != nil {
-			t.Errorf("Parser.consume: failed to parse query %q with error: %v", entry.q, err)
+			t.Errorf("parser.Parse(%s)\n= %v; want nil error", entry.q, err)
 		}
 		plnr, err := New(ctx, s, st, 0, 10, nil)
 		if err != nil {
-			t.Errorf("planner.New failed to create a valid query plan with error: %v", err)
+			t.Errorf("planner.New() = _, %v; want nil error", err)
 		}
```

Comment:
I'm assuming what is under test here is the Execute() method. If so, are `parser.Parse` and `planner.New` needed as setup for the test? If they are part of the setup: 
- Could they be extracted to a test helper?
- Can the actual system under test continue if they fail (i.e. should this be Fatalf instead of Errorf)?

Maybe it's worth adding a comment here to the reader to help them differentiate what is actually being tested. You want to make it super easy to identify what is under test and what is part of supporting that.

Since you are not comparing neither `st` nor `plnr` with anything else here, I'm going to carry on assuming they are setup. 

I know I asked you to identify function, input, output in tests in general. However, for setups that may be harder to do because you just don't want them to fail so you can carry on with your *actual* test.

Reminder: your goal here is to help the maintainer to understand what's going on. So, in this case, a helpful (and valid) error message could be:
`t.Fatalf("Failed parsing query %q: %v", entry.q, err)`
`t.Fatalf("Failed creating query plan for %q: %v", entry.q, err)`

Reply:
Thank you very much once again Tati. I totally agree and have just proceeded so.